automl/readme.md

```diff
@@ -17,24 +17,63 @@ script.
 
 ## How to install
 
-If you have [bigmler](https://bigmler.readthedocs.io/en/latest/) and
-`make` installed in your system, just check out the [WhizzML examples
+AutoML [`WhizzML Script`](./automl-script) imports a [`WhizzML
+Library`](./automl-library), so the installation is a litle bit
+different to other WhizzML scripts. You can still install it from the
+Dashboard but you will need to perform an additional step.
+
+### Using the Dashboard - 2 steps
+
+Firstly, you should install the library. Open the WhizzML Libraries
+view, as shown in the image.
+
+![Import Library](./res/import-library.png)
+
+From that view, import the library from Github, as you would do with a
+WhizzML Script, using [the link to the library ](./automl-library).
+
+![Import Library](./res/import-library2.png)
+
+Then, in the Scripts view, import the [`WhizzML
+Script`](./automl-script) from Github as you would do with other
+WhizzML scripts.
+
+![Import Script](./res/import-script.png)
+
+You may see the following error
+
+![Import Library Error](./res/lib-error.png)
+
+In that case, you will have to find, in the selector, the library you
+uploaded previously. After selecting it the error will disappear.
+
+After these steps, you will be able to import the script correctly into your
```

Comment:
Following these steps, you should be able to create the script in your BigML account.